scripts/pylint_extensions.py

```diff
@@ -1415,25 +1415,27 @@ def process_module(self, node):
         file_content = read_from_node(node)
         file_length = len(file_content)
         triple_quote_counter = 0
+        empty_line_counter = 0
         for line_num in python_utils.RANGE(file_length):
             line = file_content[line_num].strip()
             # Single line comment, ignore it.
             if line.startswith(b'#'):
                 continue
             triple_quote_counter += line.count(multi_line_indicator)
-            previous_line = file_content[line_num - 1].strip()
-            if (line.startswith(b'import') or line.startswith(b'from')):
-                if (previous_line.endswith(b'"""') and
-                        (triple_quote_counter == 2)):
+
+            if line.endswith(b'"""') and triple_quote_counter == 2:
+                temp_line_num = line_num
+                while file_content[temp_line_num + 1] == b'\n':
```

Comment:
This is careless coding. Should check that temp_line_num + 1 actually exists in the list before referencing it.

Also I suggest not doing all this logic/manipulation in the for loop -- you don't need to. Just store the variables and break, then do the next part in a separate block. 

Also, name `temp_line_num` something more clear semanitcally -- e.g. closing_line_index_of_fileoverview or similar. If you want the code to be maintained, it is important to make all of it easily understandable by fellow developers.

Reply:
done.
I'm sorry for these mistakes, I'm still learning about good coding practices in Python. I'll try to avoid these mistakes in the future.